### PR TITLE
Fix training hparams in Tacotron2

### PR DESCRIPTION
**Pull request by aaronenyeshi**
To match the correct batch size of 64 mentioned in the source code and paper.
Source: https://arxiv.org/pdf/1712.05884.pdf
Code: https://github.com/NVIDIA/tacotron2/blob/bb6761349354ee914909a42208e4820929612069/hparams.py#L84

Also, due to the larger batch size, I needed to add more .wav files (186), since the old dataloader only had 2 .wav files.

### CODE REVIEW

**Comment by aaronenyeshi** [comment]
Here are the execution results and traces:

Training Run:
image

Inference Run:
image

**Comment by aaronenyeshi** [comment]
CI is throwing `RuntimeError: CUDA out of memory.` for the following tests: 
- test_tacotron2_train_cuda
- test_tacotron2_example_cuda
- test_tacotron2_eval_cuda

I will need to disable these temporary, until we get new GPUs.

File: torchbenchmark/models/tacotron2/__init__.py [hunk 1]
```diff
@@ -19,130 +22,141 @@ def __init__(self, device=None, jit=False):
...
+                                                lr=self.train_hparams.learning_rate,
+                                                weight_decay=self.train_hparams.weight_decay)
+        self.eval_optimizer = torch.optim.Adam(self.eval_model.parameters(),
+                                               lr=self.eval_hparams.learning_rate,
+                                               weight_decay=self.eval_hparams.weight_decay)
         self.criterion = Tacotron2Loss().to(device=device)
-        train_loader, valset, collate_fn = prepare_dataloaders(self.hparams)
-        self.example_input, self.target = self.model.parse_batch(list(train_loader)[0], device=self.device)
-
+        train_loader, valset, collate_fn = prepare_dataloaders(self.train_hparams)
```
Comment (xuzhao9) [inline torchbenchmark/models/tacotron2/__init__.py:36]: Is it possible to add an option to `__init__()` that preloads the data and reprofile the workload? Would like to see if the bad GPU utilization is caused by dataloader.

**Review by xuzhao9 — APPROVED** [review]

File: torchbenchmark/models/tacotron2/__init__.py (hunk 1 above)
Comment (aaronenyeshi) [reply torchbenchmark/models/tacotron2/__init__.py:36]: Good idea, I think it can be added in a separate PR.

**Comment by aaronenyeshi** [comment]
Verified this works on A100. But disabled on T4.
```
$ gpurun python test.py --verbose -k tacotron2
srun: job 11049 queued and waiting for resources
srun: job 11049 has been allocated resources
test_tacotron2_check_device_cpu (__main__.TestBenchmark) ... skipped 'Method check_device is not implemented, skipping...'
test_tacotron2_check_device_cuda (__main__.TestBenchmark) ... ok
test_tacotron2_eval_cpu (__main__.TestBenchmark) ... skipped 'Method eval is not implemented, skipping...'
test_tacotron2_eval_cuda (__main__.TestBenchmark) ... ok
test_tacotron2_example_cpu (__main__.TestBenchmark) ... skipped 'Method get_module is not implemented, skipping...'
test_tacotron2_example_cuda (__main__.TestBenchmark) ... ok
test_tacotron2_train_cpu (__main__.TestBenchmark) ... skipped 'Method train is not implemented, skipping...'
test_tacotron2_train_cuda (__main__.TestBenchmark) ... ok

----------------------------------------------------------------------
Ran 8 tests in 87.099s

OK (skipped=4)
```

**Pull request closed without merging**